server/models/todo.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+const dateParsing = require('../helpers/dateParsing')
+module.exports = (sequelize, DataTypes) => {
+  class Todo extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  };
+  Todo.init({
+    title: DataTypes.STRING,
```

Comment:
ini validasi title ga ada ? berarti gua isi kosong gapapa ya ?

Reply:
ini sudah ada validasi not empty juga kak